iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/plan/parameter/IntoPathDescriptor.java

```diff
@@ -47,6 +49,9 @@ public class IntoPathDescriptor {
   // List<(sourceColumn, targetPath)>
   private List<Pair<String, PartialPath>> sourceTargetPathPairList;
 
+  // sourceColumn -> viewPath, if the viewPath is not empty
+  private List<Pair<String, String>> sourceColumnToViewList;
```

Comment:
It seems that `List<String> sourceColumnToViewList` is enough.

Reply:
fixed